README.md

```diff
@@ -68,6 +68,7 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
+      # if set to nil no timeout is used
```

Comment:
> We could technically ignore most of the work i've done in this PR and just document that the values are passed directly to timeout

Good point! Sorry about my initial confusion, I didn't realize that circuitbox already satisfies my request in #69.